### PR TITLE
add tests for webpack

### PR DESCRIPTION
**Pull request by dayhaysoos**
This adds the checkWeb5 and checkDwn test functions for webpack.

Not that I needed to use `/browser` for the import with webpack.

### CODE REVIEW

File: tests/webpack/src/index.js [hunk 1]
```diff
@@ -1,14 +1,48 @@
+import { Buffer } from 'buffer';
+
 import _ from 'lodash';
-import { testDwn } from './dwn-sdk-test.js'
+// import { testDwn } from './dwn-sdk-test.js'\\
```
Comment (leordev) [inline tests/webpack/src/index.js:4]: this file (and comment) can be removed, now we are running the proper tests with `checkDwn` function imported below

**Comment by dayhaysoos** [comment]
is this good to merge @leordev ?

**Comment by leordev** [comment]
> is this good to merge @leordev ?

You need to add the html elements, then yes

**Review by leordev — APPROVED** [review]
🚀

**Pull request merged**